_posts/2020-03-19-quarkus-1-3-0-final-released.adoc

```diff
@@ -0,0 +1,206 @@
+---
+layout: post
+title: 'Quarkus 1.3.0.Final released - New class loader infrastructure, GraalVM 20 support, and a lot more'
```

Comment:
```suggestion
title: 'Quarkus 1.3.0.Final released - New class loader infrastructure, GraalVM 20 support and much much more'
```

Reply:
So there is a big debate about the comma and the `and` but I don't really care :).

Reply:
I personally don't care about the comma either :P